internal/util/fscrypt/fscrypt.go

```diff
@@ -112,9 +134,11 @@ func createKeyFuncFromVolumeEncryption(
 			return nil, err
 		}
 
-		if keySize < 0 {
-			keySize = len(passphrase)
+		passphrase, err = resizePassphrase(passphrase, keySize)
+		if err != nil {
+			return nil, err
```

Comment:
can we also check wont this change break the change added in https://github.com/ceph/ceph-csi/pull/4464?